third_party/blink/renderer/core/dom/document.cc

```diff
@@ -3658,9 +3662,36 @@ void Document::close() {
   CheckCompleted();
 }
 
+void Document::RecordReplayOnRemoveLoadEventDelay() {
+  if (recordreplay::IsReplaying() &&
+      !recordreplay::FeatureEnabled("no-dependency-graph")) {
+    int node_id = V8RecordReplayDependencyGraphExecutionNode();
+    if (node_id) {
+      record_replay_load_event_dependency_nodes_.push_back(node_id);
+    } else {
+      recordreplay::Warning("DependencyGraph load event delay unknown node");
+    }
+  }
+}
+
 void Document::ImplicitClose() {
   DCHECK(!InStyleRecalc());
 
+  absl::optional<recordreplay::AutoDependencyExecution> execute;
+  if (recordreplay::IsReplaying()) {
```

Comment:
Missing that `!recordreplay::FeatureEnabled("no-dependency-graph")` check?
Could use that `ReplayAreDependenciesEnabled` function 🤷

Reply:
That isn't necessary here, the call this makes is a no-op if the dependency graph isn't enabled and the only extra compute is related to computing the JSON value.  I mainly want to avoid extra overhead while recording with these tests, I think just testing IsReplaying is fine.